logger/server.py

```diff
@@ -17,44 +17,48 @@
 import os
 import shutil
 import subprocess
+from pathlib import Path
 
 import tensorflow as tf
 
 NODE_COUNTER = {}
+BUS_COUNTER = {}
+WRITERS = {}
+LOG_DIR = "logs"
 
-async def create_new_writer(log_dir):
-    global writer
+
+async def create_new_writer(log_dir, node_name):
```

Comment:
Consider removing `async`. Function `create_new_writer` does nothing with `async/await` (does not await anything).